### PR TITLE
Fix for add hover effect in Team UI - #169

### PR DESCRIPTION
**Pull request by karagudisuma**
Changes for increased scalability for image on hover on team UI, show  social media icon, change the links for the icons accordingly.

## Description

When mouse is hovered on the cards of the /aboutus link, the card is scaled 1.6 times and the social media icons are displayed. On mouse out event they are returned to original size without social media icons.
The links to the social media icons are given and tested. 

## Motivation and Context

Solves issue described in #169.

## How Has This Been Tested?

UI changes are tested in localhost. There are no functionality change.

## Screenshots (if appropriate):
169change

# Types of changes

- [X ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: src/ignitus-CoreTeam/Styles/style.scss [hunk 1]
```diff
@@ -24,7 +24,22 @@
 }
 
 #team-avatar-card:hover {
-    transform: scale(1.2);
+    transform: scale(1.6);
```
Comment (Paarmita) [inline src/ignitus-CoreTeam/Styles/style.scss]: Please change it to 1.2 only as it looks too zoomed out at 1.6.

File: src/ignitus-CoreTeam/Styles/style.scss [hunk 2]
```diff
@@ -24,7 +24,22 @@
...
     transition-duration: 500ms;
     cursor: pointer;
+    z-index: 9999;
+    position: relative;
```
Comment (Paarmita) [inline src/ignitus-CoreTeam/Styles/style.scss]: And position should be absolute. As relative disturbs the UI on hover.

File: src/ignitus-CoreTeam/Components/Team.js [hunk 3]
```diff
@@ -8,23 +8,47 @@ import loader from "../../ignitus-Assets/Images/loader.gif";
...
+        >
           <img
             className="rounded-circle mw-100 w-75"
             src={item.img}
             alt={`avatar ${item.name}`}
           />
           <div className="title-name mt-2">{item.title}</div>
           <div className="title-description">{item.description}</div>
+          <div id={`member-${index}`} class="display-none">
+            {item.facebook && <a href={item.facebook} className="team-sm-icon-color"><i class="fa fa-facebook-square"></i></a>}
```
Comment (Paarmita) [inline src/ignitus-CoreTeam/Components/Team.js]: Social links we need to mention is only angellist and LinkedIn for all members. Can you please remove facebook and replace it with angellist along with Linkedin.

**Comment by Paarmita** [comment]
@karagudisuma Thanks for the PR. Can you please also look into this behavior of flex on hover like it comes above the navbar?

5qyec-z6248

**Comment by karagudisuma** [comment]
@Paarmita, @divyanshu-rawat  I have addressed the issues you mentioned above. The screen looks like this now:
Let me know if there is any thing I missed. 
navbar

-Thanks,
  Suma

**Comment by Paarmita** [comment]
@karagudisuma  :+1: This PR looks great - it's ready to merge! :shipit:

**Pull request merged**